static/scss/search_page.scss

```diff
@@ -0,0 +1,199 @@
+@import "../../node_modules/searchkit/theming/components";
+
+.learners-search {
+  .centered {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .fullwidth {
+    width: 100%;
+  }
+
+  // remove padding and margin from MDL layout components
+  .no-padding {
+    .mdl-grid, .mdl-cell {
+      padding: 0;
+      margin: 0;
+    }
+  }
+
+  .search-sidebar {
+    display: flex;
+    flex-direction: column;
+    background-color: $fg-grey;
+
+    .sk-panel {
+      .sk-panel__header {
+        margin: 15px;
+        font-size: 17px;
+      }
+
+      .sk-panel__content {
+        .sk-item-list > div {
+          justify-content: space-between;
+          margin-left: 25px;
+          margin-right: 40px;
+        }
+
+        .sk-item-list-option > * {
+          flex: none;
+        }
+      }
+    }
+
+    .filter-visibility-toggle.closed {
+      i.closed {
+        transform: rotate(0.75turn);
+        -webkit-transform: rotate(0.75turn);
+        -moz-transform: rotate(0.75turn);
+        -ms-transform: rotate(0.75turn);
+        -o-transform: rotate(0.75turn);
+      }
+
+      .sk-panel__content {
+        display: none;
+      }
+    }
+
+    .filter-visibility-toggle {
+      display: flex;
+      flex-direction: row;
+
+      i {
+        padding-top: 15px;
+      }
+    }
+  }
+
+  .search-sidebar * {
+    background-color: $fg-grey;
+  }
+
+  .search-header > .mdl-cell {
+    padding: 5px 0;
+  }
```

Comment:
can you not bump this definition down and indent it under `.search-header` below? if you can, you should probably put it near the `.mdl-cell` definition